README.md

```diff
@@ -1,13 +1,46 @@
+
+		
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+  <a href="mailto:sales@syncfusion.com?Subject=Syncfusion React TimePicker - Github" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples">www.syncfusion.com</a> | 1-888-9 DOTNET <br>
```

Comment:
![image](https://github.com/SyncfusionExamples/getting-started-with-the-react-timepicker/assets/95272441/31558034-7dfd-4c9f-84d0-c4fe2ab3bad7)